packages/js/components/src/experimental-select-tree-control/select-tree.tsx

```diff
@@ -69,131 +67,106 @@ export const SelectTree = function SelectTree( {
 	};
 
 	return (
-		<Dropdown
+		<div
 			className="woocommerce-experimental-select-tree-control__dropdown"
-			contentClassName="woocommerce-experimental-select-tree-control__dropdown-content"
-			focusOnMount={ false }
-			renderContent={ ( { onClose } ) =>
-				isLoading ? (
-					<div
-						style={ {
-							width: comboBoxWidth,
-						} }
-					>
-						<Spinner />
-					</div>
-				) : (
-					<Tree
-						{ ...props }
-						id={ `${ props.id }-menu` }
-						ref={ ref }
-						items={ linkedTree }
-						onTreeBlur={ onClose }
-						shouldItemBeExpanded={ shouldItemBeExpanded }
-						shouldShowCreateButton={ shouldShowCreateButton }
-						style={ {
-							width: comboBoxWidth,
-						} }
-					/>
-				)
```

Comment:
The `renderContent` content has been moved to the `SelectTreeMenu` component within the Popover.